src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2_controlnet_img2img.py

```diff
@@ -135,13 +135,11 @@ def __init__(
             scheduler=scheduler,
             movq=movq,
         )
-        movq_scale_factor = 2 ** (len(self.movq.config.block_out_channels) - 1)
-        self.image_processor = VaeImageProcessor(
-            vae_scale_factor=movq_scale_factor,
-            vae_latent_channels=self.movq.config.latent_channels,
-            resample="bicubic",
-            reducing_gap=1,
-        )
+        kwargs = {}
```

Comment:
ohh we can just give them a default value like this

```python
movq_scale_factor = 2 ** (len(self.movq.config.block_out_channels) - 1) if getattr(self, "movq", None) else ..
movq_latent_channels = self.movq.config.latent_channels if getattr(self, "movq", None) else ..
```

to be consistent with how it is handled in other pipelines, for example https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py#L218